core-rust/state-manager/src/jni/transaction_store.rs

```diff
@@ -177,13 +177,10 @@ extern "system" fn Java_com_radixdlt_transaction_REv2TransactionAndProofStore_ge
 }
 
 #[tracing::instrument(skip_all)]
-fn do_get_epoch_proof(
-    state_manager: &ActualStateManager,
-    state_version: u64,
-) -> Option<JavaLedgerProof> {
+fn do_get_epoch_proof(state_manager: &ActualStateManager, epoch: u64) -> Option<JavaLedgerProof> {
     state_manager
         .store()
-        .get_epoch_proof(state_version)
```

Comment:
_Note: pretty sure it was a typo_

Reply:
I don't think it was a typo, _epoch_ proofs are keyed by _epoch_

Reply:
yes, and the code before this PR said `state_version`. I change it to `epoch` in this PR.